addons/MetroidvaniaSystem/Database/MapEditor.tscn

```diff
@@ -1131,6 +1160,7 @@ grow_horizontal = 2
 grow_vertical = 2
 focus_mode = 2
 mouse_filter = 1
+script = SubResource("GDScript_j1vk0")
```

Comment:
Instead of adding this script, it would be simpler to use `set_drag_forwarding()` and handle the methods in `MapEditor.gd`.

Reply:
Updated.